hokusai/commands/setup.py

```diff
@@ -6,17 +6,22 @@
 
 import yaml
 
-from jinja2 import Environment, PackageLoader
-env = Environment(loader=PackageLoader('hokusai', 'templates'))
+from jinja2 import Environment, PackageLoader, FileSystemLoader
 
 from hokusai.lib.command import command
 from hokusai.lib.config import config
 from hokusai.services.ecr import ECR
-from hokusai.lib.common import print_green, build_service, build_deployment, YAML_HEADER
+from hokusai.lib.common import print_green, YAML_HEADER
 from hokusai.lib.exceptions import HokusaiError
 
 @command
-def setup(aws_account_id, project_type, project_name, aws_ecr_region, port, internal):
+def setup(aws_account_id, project_type, project_name, aws_ecr_region, port, internal, template_dir):
+  if template_dir:
+    env = Environment(loader=FileSystemLoader(template_dir))
+  else:
+    env = Environment(loader=PackageLoader('hokusai', 'templates'))
+
+  print("THIS IS THE CORRECT VERSION STILL")
```

Comment:
@saolsen lol debug?

Reply:
this is a feature, the software reassures you that everything is fine